src/erlcloud_aws.erl

```diff
@@ -698,7 +698,10 @@ service_config( <<"athena">> = Service, Region, Config ) ->
 service_config(<<"cloudwatch_logs">>, Region, Config)->
     Host = service_host(<<"logs">>, Region),
     Config#aws_config{cloudwatch_logs_host = Host};
-service_config( <<"waf">>, _Region, Config ) -> Config.
+service_config( <<"waf">>, _Region, Config ) -> Config;
```

Comment:
time to refactor this fun...as why does not WAF has the service_host call?

Reply:
As I understand, `waf` endpoint does not include region. e.g. `waf.amazonaws.com`